src/main/java/org/spongepowered/api/service/pagination/PaginationList.java

```diff
@@ -202,26 +246,34 @@ default void sendTo(MessageChannel channel) {
         PaginationList build();
 
         /**
-         * Send the constructed pagination list to the given receiver.
+         * Sends the constructed pagination list to the given receiver.
          *
-         * @param receiver The receiver to send to
+         * @param receiver The receiver to send the list to
          */
         default void sendTo(MessageReceiver receiver) {
             build().sendTo(receiver);
         }
 
         /**
-         * Send the constructed pagination list to the specified
-         * {@link MessageChannel}.
+         * Sends the constructed pagination list to the members of the
+         * specified {@link MessageChannel}.
          *
-         * @param channel channel to send to
+         * @param channel The channel to send the list to
          */
         default void sendTo(MessageChannel channel) {
-            channel.getMembers().forEach(this::sendTo);
+            final PaginationList list = build();
+            channel.getMembers().forEach(list::sendTo);
         }
 
+        /**
+         * Sends the constructed pagination list to an iterable of
+         * {@link MessageReceiver}s.
+         *
+         * @param receivers The message receivers to send the list to
+         */
         default void sendTo(Iterable<MessageReceiver> receivers) {
-            receivers.forEach(this::sendTo);
+            final PaginationList list = build();
+            receivers.forEach(list::sendTo);
```

Comment:
Maybe change it to the plain for each loop as that is usually faster (no need to create the lambda object).